src/main/java/com/webonise/todoapp/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.webonise.todoapp.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import com.webonise.todoapp.model.User;
+import com.webonise.todoapp.service.UserService;
+
+@RestController
+@CrossOrigin(origins = "*", allowedHeaders = "*")
+
+public class UserController {
+	@Autowired
+	private UserService userService;
+
+	@PostMapping("/logIn")
+	public Boolean logIn(@RequestBody User user) {
+		return userService.verifyUser(user);
```

Comment:
in case of a user being valid you will just return true?How will session management be done?